lib/reports/github_api_client.rb

```diff
@@ -17,15 +17,29 @@ class AuthenticationFailure < Error; end
   class NonexistentUser < Error; end
   class RequestFailure < Error; end
   class ConfigurationError < Error; end
+  class GistCreationFailure < Error; end
 
   User = Struct.new(:name, :location, :public_repos)
   Repo = Struct.new(:name, :languages)
   Event = Struct.new(:type, :repo_name)
+  Gist = Struct.new(:url)
+  Star = Struct.new(:owner, :repo)
 
   class GitHubAPIClient
+    def initialize
+      level = ENV["LOG_LEVEL"]
+      @logger = Logger.new(STDOUT)
+
+      @logger.formatter = proc do |severity, datetime, program, message|
```

Comment:
Unused block argument - `severity`. If it's necessary, use `_` or `_severity` as an argument name to indicate that it won't be used.<br>Unused block argument - `datetime`. If it's necessary, use `_` or `_datetime` as an argument name to indicate that it won't be used.<br>Unused block argument - `program`. If it's necessary, use `_` or `_program` as an argument name to indicate that it won't be used.